tests/other_test.go

```diff
@@ -16,9 +16,8 @@ func TestRunBasicExample(t *testing.T) {
 	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
 		Testing:      t,
 		TerraformDir: basicExampleTerraformDir,
-		Prefix:       "s2s-basic",
 		TerraformVars: map[string]interface{}{
-			"prefix": options.Prefix,
+			"prefix": "s2s-basic",
```

Comment:
why are we hard coding a prefix here and not using the wrapper Prefix option to generate a unique for for every test?

Reply:
I had forgotten about the random string being added by the wrapper, updated.